### PR TITLE
doc/cephadm: accessing the admin socket

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/cephadm/troubleshooting.rst [hunk 1]
```diff
@@ -161,3 +161,15 @@ This means that you must run a command of this form::
   ceph config set mon public_network <mon_network>
 
 For more detail on operations of this kind, see :ref:`deploy_additional_monitors`
+
+Accessing the admin socket
+--------------------------
+
+Each Ceph daemon provides an admin socket bypassing the
+MONs (See :ref:`rados-monitoring-using-admin-socket`).
```
Comment (asettle) [inline doc/cephadm/troubleshooting.rst:169]: Does this mean "Each Ceph daemon provides an admin socket **that** bypasses the Mons" or "Each Ceph daemon provides an admin socket **by** bypassing the MONs"?

File: doc/cephadm/troubleshooting.rst [hunk 2]
```diff
@@ -161,3 +161,15 @@ This means that you must run a command of this form::
...
+
+As the daemons run within containers, we first have to enter the container on
```
Comment (asettle) [inline doc/cephadm/troubleshooting.rst:171]:
```suggestion
As the daemons run within containers, enter the container on

File: doc/cephadm/troubleshooting.rst (hunk 1 above)
Comment (mgfritch) [inline doc/cephadm/troubleshooting.rst:169]: > Does this mean "Each Ceph daemon provides an admin socket **that** bypasses the Mons" or "Each Ceph daemon provides an admin socket **by** bypassing the MONs"?


**that** 

```suggestion
Each Ceph daemon provides an admin socket that bypass the
MONs (See :ref:`rados-monitoring-using-admin-socket`).
```

Comment (sebastian-philipp) [reply doc/cephadm/troubleshooting.rst:169]: the former

File: doc/cephadm/troubleshooting.rst [hunk 3]
```diff
@@ -161,3 +161,15 @@ This means that you must run a command of this form::
...
+the host before accessing the admin socket::
```
Comment (mgfritch) [inline doc/cephadm/troubleshooting.rst:172]:
```suggestion
To access the admin socket, first enter the daemon container on the host::
```
In addition to the suggestion from @asettle ... also drop the phrase `As daemons run within containers` ?

File: doc/cephadm/troubleshooting.rst [hunk 4]
```diff
@@ -161,3 +161,15 @@ This means that you must run a command of this form::
   ceph config set mon public_network <mon_network>
 
 For more detail on operations of this kind, see :ref:`deploy_additional_monitors`
+
+Accessing the admin socket
+--------------------------
+
+Each Ceph daemon provides an admin socket bypassing the
```
Comment (asettle) [inline doc/cephadm/troubleshooting.rst:168]:
```suggestion
Each Ceph daemon provides an admin socket that bypasses the

File: doc/cephadm/troubleshooting.rst (hunk 1 above)
Comment (asettle) [reply doc/cephadm/troubleshooting.rst:169]: ADded suggestion for ease :)

**Comment by sebastian-philipp** [comment]
Did I said that I'm not the best person to add doc PRs?

**Pull request merged**